app/views/topical_events/show.html.erb

```diff
@@ -58,7 +58,9 @@
 
     <% if @topical_event.image_url %>
       <div class="topical-events__image">
-      <%= image_tag(@topical_event.image_url, alt: @topical_event.image_alt_text, srcset: { @topical_event.image_srcset[2][:url] => "2x", @topical_event.image_srcset[0][:url] => "3x",}) %>
+      <%= @topical_event.image_srcset.any? ?
```

Comment:
Ah. I missed that you added the conditional as a ternary operator, hence my previous question about what happens if `image_srcset` is `nil`. I've never seen a ternary operator written over multiple lines like this before. 

Given that logic in views is generally more difficult to read, I'd recommend using a standard if/else here instead.

Reply:
fixed this